src/PIL/ImageOps.py

```diff
@@ -610,6 +611,10 @@ def exif_transpose(image, *, in_place=False):
         exif = exif_image.getexif()
         if ExifTags.Base.Orientation in exif:
             del exif[ExifTags.Base.Orientation]
+            if in_place and ExifTags.Base.Orientation in getattr(
+                exif_image, "tag_v2", {}
```

Comment:
Isn't this format-specific? Probably move it to `load_end`?

Reply:
Oh, because `load()` is always run by the time we get here. Yep, good call, I've pushed a commit for that.